storage/range.go

```diff
@@ -2028,6 +2043,7 @@ func (r *Range) AdminMerge(args *proto.AdminMergeRequest, reply *proto.AdminMerg
 					UpdatedDesc:    updatedDesc,
 					SubsumedRaftID: subsumedDesc.RaftID,
 				},
+				Intents: []proto.Key{desc1Key, desc2Key},
```

Comment:
desc keys are in the system key local area and since both ranges in a merge must be on the same store, they're guaranteed to be available to resolve as part of the end transaction.